etc/avocado/avocado.conf

```diff
@@ -1,10 +1,12 @@
 [datadir.paths]
 # Avocado data dir (holds tests and test auxiliary data, such as ISO files).
-base_dir = /var/lib/avocado
+#base_dir = /var/lib/avocado
+base_dir = /home/medic/avocado/data
 # You may override the specific test directory with test_dir
 test_dir = /usr/share/avocado/tests
 # You may override the specific test auxiliary data directory with data_dir
-data_dir = /var/lib/avocado/data
+#data_dir = /var/lib/avocado/data
```

Comment:
omg, how did I missed that... (btw I used it to share `avocado-vt` data with root 😉) I'll put these in user-local-config to avoid it in the future...